Simplified/NYPLBarcode.swift

```diff
@@ -55,7 +55,7 @@ final class NYPLBarcode: NSObject {
 
   class func presentScanner(withCompletion completion: @escaping (String?) -> ())
   {
-    AVCaptureDevice.requestAccess(forMediaType: AVMediaTypeVideo) { granted in
+    AVCaptureDevice.requestAccess(for: AVMediaType(rawValue: convertFromAVMediaType(AVMediaType.video))) { granted in
```

Comment:
I think this can all be rewritten as:
`AVCaptureDevice.requestAccess(for: .video) { granted in`
Let me know if that works.